### PR TITLE
Support Update VS Code from Command Palette

### LINKED ISSUE #77184: Update VS Code from Command Palette
Ability to update VS Code from the command palette as I have disabled activity bar and have to reenable it to update VS Code.
Also, there is a palette option to check for extensions updates, but not for the whole app

Steps:
1. Open Command palette
2. Input `>Check for VS Code Update`
3. Update VS Code

### PR DESCRIPTION
**Pull request by DiamondYuan**
close https://github.com/microsoft/vscode/issues/77184

### CODE REVIEW

**Comment by DiamondYuan** [comment]
@joaomoreno 

CI broken,How to fix it ?

```
src/vs/workbench/contrib/update/common/update.ts:6:48:Unexternalized string found: "update.showCurrentReleaseNotes"
src/vs/workbench/contrib/update/common/update.ts:7:45:Unexternalized string found: "update.checkForVSCodeUpdate"
```

**Review by joaomoreno — CHANGES_REQUESTED** [review]
Please merge both classes, there's no need for an abstract class here.

File: src/vs/workbench/contrib/update/common/update.ts [hunk 1]
```diff
@@ -3,4 +3,5 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-export const ShowCurrentReleaseNotesActionId = 'update.showCurrentReleaseNotes';
\ No newline at end of file
+export const ShowCurrentReleaseNotesActionId = "update.showCurrentReleaseNotes";
+export const CheckForVSCodeUpdateActionId = "update.checkForVSCodeUpdate";
```
Comment (joaomoreno) [inline src/vs/workbench/contrib/update/common/update.ts]: You need to use single quotes, not double quotes.

**Review by joaomoreno — CHANGES_REQUESTED** [review]

**Comment by DiamondYuan** [comment]
> Please merge both classes, there's no need for an abstract class here.

Ci Broken,Time out

**Comment by joaomoreno** [comment]
CI is not broken. You are using double quotes instead of double quotes. CI is telling you exactly that. Fix that please.

**Comment by DiamondYuan** [comment]
> CI is not broken. You are using double quotes instead of double quotes. CI is telling you exactly that.

image


https://dev.azure.com/vscode/VSCode/_build/results?buildId=23300

Last CI broken was due to timeout. I have fix the error.

**Comment by DiamondYuan** [comment]
cc @joaomoreno  when will whit PR be merged ?

**Review by ian-sawyer — APPROVED** [review]

**Comment by joaomoreno** [comment]
Thanks! :beers:

**Pull request merged**